PowerValidatedSolutions.psm1

```diff
@@ -51698,6 +51698,70 @@ Function New-AriaNetworksNsxtDataSource {
     }
 }
 Export-ModuleMember -Function New-AriaNetworksNsxtDataSource
+
+Function Update-AriaNetworksvCenterDataSourceCredentials {
+    <#
+        .SYNOPSIS
+        Update a configured vCenter Server data source's credentials in VMware Aria Operations for Networks.
+
+        .DESCRIPTION
+        The Update-AriaNetworksvCenterDataSourceCredentials cmdlet allows a user to update a previously configured vCenter Servers credentials in VMware Aria Operations for Networks.
+
+        .EXAMPLE
+        Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1!  -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
```

Comment:
```suggestion
        Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
```

Reply:
Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9